pkgs/by-name/li/libgrapheme/package.nix

```diff
@@ -9,6 +9,11 @@ stdenv.mkDerivation rec {
     hash = "sha256-pou93edr1Vul1kEWzl5CoT3wRcgcCFLemrYIlqoUMSU=";
   };
 
+  postPatch = ''
+    substituteInPlace configure \
+      --replace-fail "uname" "echo ${stdenv.hostPlatform.uname.system}"
```

Comment:
You could use `deterministic-host-uname`

Reply:
But deterministic-host-uname returns build host name, not a target host name.

Reply:
True, I left a comment in https://github.com/NixOS/nixpkgs/pull/310385/files#r1755589455

For now you can do `(deterministic-uname.override { forPlatform = stdenv.hostPlatform; })`

Reply:
Well, `nativeBuildInputs = [ which (deterministic-uname.override { forPlatform = stdenv.hostPlatform; }) ];` doesn't work anyway - `uname` picked from `coreutils`.

Reply:
Okay, let's leave it like this since there's no multiple uname runs.